fishtank.py

```diff
@@ -48,6 +48,14 @@ def __init__(self, x, y):
         self.rect.x = x
         self.rect.y = y
 
+    def __lt__(self, other):
+        if self.rect.x < other.rect.x:
+            return True
+        elif self.rect.y < other.rect.y:
+            return True
+        else:
+            return False
+
```

Comment:
This change was necessary for sorting collections of Coins. The "lt" in __lt__ stands for "less than" and is used for comparing the value of objects,